integration/internal/main.go

```diff
@@ -0,0 +1,20 @@
+package internal
```

Comment:
nit: I find "internal" not very descriptive. "helpers" or "utils" or "setuputil" would be better,

Reply:
I can move it to a package inside `internal`. We do need the `internal` nesting because it means it can only be imported by other packages inside `integration`. See https://go.dev/doc/go1.4#internalpackages